setup.py

```diff
@@ -0,0 +1,27 @@
+import setuptools
+
+with open('README.md', 'r') as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name='secrets-management',
+    version='0.0.2',
+    author='Beauhurst',
+    author_email='noreply@beauhurst.com',
+    description='Helper library for interacting with AWS Secrets Manager',
+    long_description=long_description,
+    long_description_content_type='text/markdown',
+    url='https://github.com/beauhurst/secrets-management',
+    packages=setuptools.find_packages(),
+    classifiers=[
+        'Programming Language :: Python :: 3',
+        'License :: OSI Approved :: MIT License',
+        'Operating System :: OS Independent',
+    ],
+    python_requires='>=3.8',
```

Comment:
Would need to this to be 3.6+ to support, for example, F2

Reply:
fixed